src/istio/utils/attribute_names.cc

```diff
@@ -28,6 +28,8 @@ const char AttributeName::kRequestMethod[] = "request.method";
 const char AttributeName::kRequestPath[] = "request.path";
 const char AttributeName::kRequestReferer[] = "request.referer";
 const char AttributeName::kRequestScheme[] = "request.scheme";
+const char AttributeName::kRequestUrlPath[] = "request.url_path";
```

Comment:
have these attributes been added to the global dictionary in istio/api ?

Reply:
Just added: https://github.com/istio/api/pull/566